loky/process_executor.py

```diff
@@ -874,6 +874,9 @@ def __init__(self, max_workers=None, job_reducers=None,
                 object should provide SimpleQueue, Queue and Process.
             initializer: An callable used to initialize worker processes.
             initargs: A tuple of arguments to pass to the initializer.
+            env: A dict of environment variable to overwrite in the child
+                process. This only works with the loky context. The environment
+                variables are set before any module load.
```

Comment:
Maybe add a comment stating that this does not work reliably under Python 2.7 with Windows.